cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaConnectClusterTest.java

```diff
@@ -165,12 +167,15 @@ public class KafkaConnectClusterTest {
 
     @ParallelTest
     public void testMetricsConfigMap() {
-        ConfigMap metricsCm = kc.generateMetricsAndLogConfigMap(new MetricsAndLogging(metricsCM, null));
-        checkMetricsConfigMap(metricsCm);
+        ConfigMap configMap = kc.generateConnectConfigMap(new MetricsAndLogging(metricsCM, null));
+        checkMetricsConfigMap(configMap);
     }
 
-    private void checkMetricsConfigMap(ConfigMap metricsCm) {
-        assertThat(metricsCm.getData().get(MetricsModel.CONFIG_MAP_KEY), is(metricsCmJson));
+    private void checkMetricsConfigMap(ConfigMap configMap) {
```

Comment:
```suggestion
    private void checkConnectConfigMap(ConfigMap configMap) {
```
Looks like this method is only used by one test so it could probably be collapsed into that test directly